extension/src/setup.js

```diff
@@ -3,6 +3,18 @@ const ctpClientBuilder = require('./ctp')
 const logger = require('./utils').getLogger()
 const { ensureResources } = require('./config/init/ensure-resources')
 
+function _generateAuthorizationHeaderValue(ctpProjectKey) {
+  const ctpConfig = config.getCtpConfig(ctpProjectKey)
+  if (ctpConfig && ctpConfig.username && ctpConfig.password) {
```

Comment:
Again you can use optional chaining if you want 😄  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
```suggestion
  if (ctpConfig?.username && ctpConfig?.password) {
```

Reply:
Optionl chaining is only compatible with node v.14 or above. Shall we change to it?

Reply:
Yes, we require explicitly node v14 or higher :) https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/package.json#L40

Reply:
Moved to another class already and changed it to optional chaining
https://github.com/commercetools/commercetools-adyen-integration/commit/5e5c59f61fe582a6268efc3944d24cc579da511a